sycl-fusion/jit-compiler/lib/translation/KernelTranslation.cpp

```diff
@@ -46,7 +46,7 @@ static const char *WORK_GROUP_SIZE_HINT_ATTR = "work_group_size_hint";
 ///   - reqd_work_group_size
 ///   - work_group_size_hint
 static void restoreKernelAttributes(Module *Mod, SYCLKernelInfo &Info) {
-  auto *KernelFunction = Mod->getFunction(Info.Name);
+  auto *KernelFunction = Mod->getFunction(StringRef{Info.Name});
```

Comment:
Question: is the explicit constructor needed?

Reply:
Neat, I think it was needed to get the implicit conversion from `DynString` to `const char*`, but now that that is gone, we can rely on the `StringRef'` implicit constructors.